executor/common.h

```diff
@@ -55,20 +55,16 @@ extern call_t syscalls[];
 extern unsigned syscall_count;
 #endif
 
-#if defined(SYZ_EXECUTOR) || (defined(SYZ_REPEAT) && defined(SYZ_WAIT_REPEAT)) ||            \
-    defined(SYZ_USE_TMP_DIR) || defined(SYZ_TUN_ENABLE) || defined(SYZ_SANDBOX_NAMESPACE) || \
-    defined(SYZ_SANDBOX_SETUID) || defined(SYZ_FAULT_INJECTION) || defined(__NR_syz_kvm_setup_cpu)
-const int kFailStatus = 67;
-const int kRetryStatus = 69;
-#endif
-
 #if defined(SYZ_EXECUTOR)
 const int kErrorStatus = 68;
 #endif
 
 #if defined(SYZ_EXECUTOR) || (defined(SYZ_REPEAT) && defined(SYZ_WAIT_REPEAT)) ||            \
     defined(SYZ_USE_TMP_DIR) || defined(SYZ_TUN_ENABLE) || defined(SYZ_SANDBOX_NAMESPACE) || \
     defined(SYZ_SANDBOX_NONE) || defined(SYZ_SANDBOX_SETUID) || defined(__NR_syz_kvm_setup_cpu)
+const int kFailStatus = 67;
+const int kRetryStatus = 69;
```

Comment:
We seem to need kRetryStatus in exitf. exitf is enabled if SYZ_FAULT_INJECTION, but kRetryStatus is not provided if SYZ_FAULT_INJECTION. Will it fail to build? Why wan't it caught by tests?

Reply:
Done. See reply above regarding tests.